src/resolve/resolvectl.c

```diff
@@ -1154,6 +1160,20 @@ static int show_statistics(int argc, char **argv, void *userdata) {
                            TABLE_FIELD, "Cache Misses",
                            TABLE_UINT64, n_cache_miss,
                            TABLE_EMPTY, TABLE_EMPTY,
+                           TABLE_STRING, "Serve Stale",
```

Comment:
hmm, is that an appropriate title? this covers failure statistics regardless ot the "stale" cache thing or not. Just call this "Failure Statistics"?